modules/dynamic-provisioning-azure-disk-definition.adoc

```diff
@@ -18,25 +18,25 @@ parameters:
   storageaccounttype: Standard_LRS  <2>
   kind: Dedicated  <3>
 ----
-<1> Azure storage account name. This must reside in the same resource 
-group as the cluster. If a storage account is specified, the `location` 
-is ignored. If a storage account is not specified, a new storage 
-account gets created in the same resource group as the cluster. If you 
+<1> Azure storage account name. This must reside in the same resource
+group as the cluster. If a storage account is specified, the `location`
+is ignored. If a storage account is not specified, a new storage
+account gets created in the same resource group as the cluster. If you
 are specifying a `storageAccount`, the value for `kind` must be `Dedicated`.
-<2> Azure storage account SKU tier. Default is empty. Note that Premium 
-VMs can attach both _Standard_LRS_ and _Premium_LRS_ disks, Standard VMs 
-can only attach _Standard_LRS_ disks, Managed VMs can only attach 
+<2> Azure storage account SKU tier. Default is empty. Note that Premium
```

Comment:
"Default is empty." What does this mean? What happens if you don't specify a value?